### PR TITLE
Add karl-police to contributor list

### PR DESCRIPTION
**Pull request by karl-police**
Pull request checklist:

- [ ] `CHANGELOG.md` was updated, if applicable

### CODE REVIEW

**Comment by karl-police** [comment]
Wait... did the format thing actually break?

I managed to build this in Visual Studio fine.

**Comment by Mm2PL** [comment]
Rename your avatar file (and the reference in the contributors file) to have an underscore instead of `-`. C++ is interpretting your name as karl subtract police.

**Comment by karl-police** [comment]
> Rename your avatar file (and the reference in the contributors file) to have an underscore instead of `-`. C++ is interpretting your name as karl subtract police.

Does the auto generator, name the variables based on filename?

**Comment by Mm2PL** [comment]
https://github.com/Chatterino/chatterino2/blob/master/resources/generate_resources.py#L51
looks like it

**Comment by karl-police** [comment]
> Wait... did the format thing actually break?
> 
> I managed to build this in Visual Studio just fine.

I wonder how VS didn't complain about it

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Removed timestamp from AutoMod messages. (#3503)
```
Comment (Felanbird) [inline CHANGELOG.md:5]:
```suggestion
```
Old and Duplicate changelog.

Comment (karl-police) [reply CHANGELOG.md:5]: Yeah, i don't know how I can remove these from the branch.

Comment (LosFarmosCTL) [reply CHANGELOG.md:5]: > Yeah, i don't know how I can remove these from the branch.

By deleting them in the file and pushing those changes??

Comment (ALazyMeme) [reply CHANGELOG.md:5]: > Yeah, i don't know how I can remove these from the branch.

You click the button on felanbird's comment to merge the change he suggested.

**Review by Felanbird — APPROVED** [review]
lgtm

**Pull request merged**